_src/static/sass/utilities/_variables.scss

```diff
@@ -3,156 +3,316 @@
-$gray-lighter:     tint($gray, 50%);
-$gray-lightest:    tint($gray, 75%);
-$gray-dark:        shade($gray, 20%);
-$gray-darker:      shade($gray, 50%);
-$gray-darkest:     shade($gray, 75%);
-
-$blue-light:       tint($blue, 20%);
-$blue-lighter:     tint($blue, 50%);
-$blue-lightest:    tint($blue, 75%);
-$blue-dark:        shade($blue, 20%);
-$blue-darker:      shade($blue, 50%);
-$blue-darkest:     shade($blue, 75%);
-
-$pink-light:       tint($pink, 20%);
-$pink-lighter:     tint($pink, 50%);
-$pink-lightest:    tint($pink, 75%);
-$pink-dark:        shade($pink, 20%);
-$pink-darker:      shade($pink, 50%);
-$pink-darkest:     shade($pink, 75%);
-
-$red-light:       tint($red, 20%);
-$red-lighter:     tint($red, 50%);
-$red-lightest:    tint($red, 75%);
-$red-dark:        shade($red, 20%);
-$red-darker:      shade($red, 50%);
-$red-darkest:     shade($red, 75%);
-
-$green-light:       tint($green, 20%);
-$green-lighter:     tint($green, 50%);
-$green-lightest:    tint($green, 75%);
-$green-dark:        shade($green, 20%);
-$green-darker:      shade($green, 50%);
-$green-darkest:     shade($green, 75%);
-
-$yellow-light:       tint($yellow, 20%);
-$yellow-lighter:     tint($yellow, 50%);
-$yellow-lightest:    tint($yellow, 75%);
-$yellow-dark:        shade($yellow, 20%);
-$yellow-darker:      shade($yellow, 50%);
-$yellow-darkest:     shade($yellow, 75%);
-
-$orange-light:       tint($orange, 20%);
-$orange-lighter:     tint($orange, 50%);
-$orange-lightest:    tint($orange, 75%);
-$orange-dark:        shade($orange, 20%);
-$orange-darker:      shade($orange, 50%);
-$orange-darkest:     shade($orange, 75%);
 
+// typeface imports (TODO: these should be abstracted out once we have more than just 1)
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-Light-webfont', 300, $file-formats: eot woff ttf svg);
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-LightItalic-webfont', 300, italic, $file-formats: eot woff ttf svg);
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-Regular-webfont', 400, $file-formats: eot woff ttf svg);
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-Italic-webfont', 400, italic, $file-formats: eot woff ttf svg);
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-Semibold-webfont', 600, $file-formats: eot woff ttf svg);
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-SemiboldItalic-webfont', 600, italic, $file-formats: eot woff ttf svg);
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-Bold-webfont', 700, $file-formats: eot woff ttf svg);
+@include font-face('Open Sans', '../fonts/OpenSans/OpenSans-BoldItalic-webfont', 700, italic, $file-formats: eot woff ttf svg);
+
+// font families
+$font-family-sans-serif: 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
+$font-family-monospace: 'Bitstream Vera Sans Mono', Consolas, Courier, monospace;
+
+// scale roots
+$font-scale-small: 14;
+$font-scale-medium: 16;
+$font-scale-large: 16;
```

Comment:
yes, these are roots that will be used for different breakpoint/response font bases - the scale will flex depending on this initial value. So, medium will be used for medium-based viewport sizes and large will be used for large.